perun-core/src/main/java/cz/metacentrum/perun/core/api/MembersManager.java

```diff
@@ -1152,6 +1152,11 @@
 	List<RichMember> getSponsoredMembers(PerunSession sess, Vo vo, User user) throws InternalErrorException, PrivilegeException;
 
 	/**
+	 * Gets list of sponsored members of a VO.
+	 */
```

Comment:
Please extend the javadoc - add \@ param, \@ return and \@ throws annotations. All of the methods should have this. 
I see that some of other methods don't have this. I will contact the author and ask him to fix it.